### PR TITLE
Strip off leading/trailing whitespace from imports

### PR DESCRIPTION
**Pull request by WorkSutton**
Clean up imports from spreadsheets

### CODE REVIEW

**Comment by go8soft** [comment]
@WorkSutton we dont need this PR, because of https://github.com/wearefuturegov/beacon/pull/36 and https://apidock.com/rails/ActiveRecord/Callbacks/before_save. Because we are using Contact object it will take care for the strip.

**Comment by WorkSutton** [comment]
Ah good catch @go8soft. I had created this first but didn't push it up. I wanted this as a belt and braces incase we did not go ahead with the other PR.

File: app/models/imported_item.rb [hunk 1]
```diff
@@ -22,17 +22,17 @@ def import(params)
...
+        nhs_number: row[1].strip,
         is_vulnerable: (row[2] == 1 || row[2].to_s.downcase == 'true'),
-        first_name: row[3],
-        surname: row[4],
+        first_name: row[3].strip,
+        surname: row[4].strip,
         date_of_birth: row[5],
-        email: row[6],
+        email: row[6].strip,
         mobile: row[7],
```
Comment (dmgarland) [inline app/models/imported_item.rb:32]: Why are some fields stripped and not others?

Comment (WorkSutton) [reply app/models/imported_item.rb:32]: @dmgarland I targeted text/string fields. Basically ones that will respond to strip

**Comment by WorkSutton** [comment]
Given @go8soft has correctly identified that if my other [PR 363](https://github.com/wearefuturegov/beacon/pull/363) is approved by all, and it will mean that imported data is saved correctly too, should we leave this as a belt and braces, or just drop it?

**Comment by WorkSutton** [comment]
Closed as functionality will be delivered in PR363

**Pull request closed without merging**